### ISSUE TITLE
Preact X children.filter is not a function

### ISSUE THREAD

**Issue by brielov** [developit/preact-router#299]
I'm testing preact-router with preact X (the recent alpha release) and this error comes up with a very basic app.

```javascript
import { h } from 'preact';
import { Router } from 'preact-router';
import Home from './components/Home';

export default function App() {
  return (
    <Router>
      <Home path="/" />
    </Router>
  );
}
```

And the error:

```
Uncaught TypeError: children.filter is not a function
    at Router.getMatchingChildren (index.js:213)
    at Router.render (index.js:231)
    at _ (index.js:126)
    at _ (index.js:137)
    at d (children.js:74)
    at $ (render.js:17)
    at Object.parcelRequire.index.js.preact (index.js:6)
    at newRequire (src.e31bb0bc.js:49)
    at src.e31bb0bc.js:81
    at src.e31bb0bc.js:107
```

Am I doing something wrong or is it a known issue with preact X ?

**Comment by developit** [comment]
Known issue - Router needs to be updated.

**Comment by developit** [comment]
Fixed in #298 by @sync!
